modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -562,6 +562,12 @@ public ObjectEntry deleteObjectEntry(ObjectEntry objectEntry)
 			Collections.emptyMap(), objectDefinition.getObjectDefinitionId(),
 			objectEntry::getValues);
 
+		_friendlyURLEntryLocalService.deleteFriendlyURLEntry(
+			objectEntry.getNonzeroGroupId(),
+			_classNameLocalService.getClassNameId(
+				objectDefinition.getClassName()),
+			objectEntry.getObjectEntryId());
```

Comment:
Does the order matter?

Can we move this to before `_workflowInstanceLinkLocalService.deleteWorkflowInstanceLinks(`) ?
I just thought about putting together the `*LocalService` calls and ordering them. 
It would be a guess to organize it for Brian.

Reply:
I changed as requested, but now `_addFriendlyURLEntry` is after the workflow code and  `deleteFriendlyURLEntry` is before the workflow code 👀

Reply:
:grimacing: :crossed_fingers: :pray: